code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -467,4 +497,76 @@
 	set category = "Object"
 	set src in usr
 
-	AltClick(usr)
\ No newline at end of file
+	AltClick(usr)
+
+
+/obj/item/weapon/material/twohanded/pike
+	icon_state = "pike0"
+	base_icon = "pike"
+	name = "pike"
+	desc = "A long spear used by the infantry in ancient times."
+	force = 5
+	unwielded_force_divisor = 0.2
+	force_divisor = 0.3
+	edge = 1
+	w_class = 4.0
+	slot_flags = SLOT_BACK
+	attack_verb = list("attacked", "poked", "jabbed", "gored", "stabbed")
+	default_material = "steel"
+	reach = 2
+	applies_material_colour = 0
+	can_embed = 0
+
+/obj/item/weapon/material/twohanded/pike/halberd
+	icon_state = "alberd0"

```

Comment:
Is this a typo? Shouldn't it be halberd?

Reply:
It is a typo, it will be fixed.